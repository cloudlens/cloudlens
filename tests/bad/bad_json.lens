stream when(e) {
    print("foo")
}
