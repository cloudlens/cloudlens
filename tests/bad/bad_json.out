SyntaxError: Invalid JSON: <json>:1:0 Expected json literal but found ident
foo barbar foo
^ in <eval> at line number 1
